### PR TITLE
Update conveners for 2022

### PR DESCRIPTION
**Pull request by graeme-a-stewart**
Make sure conveners list all the right people for 2022.
Light consistency update for all working groups to ensure
that both experiment and affiliation are listed.

### CODE REVIEW

File: _workinggroups/frameworks.md [hunk 1]
```diff
@@ -46,8 +46,14 @@ discussions are held on the [hsf-tech-forum](https://groups.google.com/forum/#%2
 
 The group conveners are:
 
-- Chris Jones, FNAL <cdj@fnal.gov>
-- Kyle Knoepfel, FNAL <knoepfel@fnal.gov>
-- Attila Krasznahorkay, CERN <Attila.Krasznahorkay@cern.ch>
+- Benedikt Hegner, CERN and FCC
```
Comment (hegner) [inline _workinggroups/frameworks.md:49]: Key4HEP. Not doing much FCC these days

File: _workinggroups/generators.md [hunk 2]
```diff
@@ -62,14 +62,15 @@ on [HSF/documents](https://github.com/HSF/documents/tree/master/CWP/papers/HSF-C
 
 #### Current convenors
 
-- Andrea Valassi (2018-2021)
-- Efe Yazgan (2020-2021)
-- Josh McFayden (2018-2021)
+- Markus Deifenthaler, EIC and JLab (2022)
```
Comment (hegner) [inline _workinggroups/generators.md:65]: Diefenthaler ;-)

File: _workinggroups/pyhep.md [hunk 3]
```diff
@@ -191,3 +192,4 @@ The event was kindly sponsored by
 
 - Chris Tunnell (XENON1T), 2019
 - Graeme Stewart (CERN EP-SFT), 2018
+- Ben Kirkler (LZ, CMS) 2020
```
Comment (hegner) [inline _workinggroups/pyhep.md:195]: Krikler, no?

**Comment by graeme-a-stewart** [comment]
All the link check failures seem to be the same kind as yesterday and all false positives. Maybe @klieret has an idea - it seems that `{{site.baseurl}}` is expanding to an empty string and then the URL checker is trying for an absolute path, instead of a relative one.

File: _workinggroups/training.md [hunk 4]
```diff
@@ -145,24 +145,23 @@ We are always looking for volunteers from the community to help us with our trai
...
 Upcoming schools:
 
 {% include list_of_upcoming_schools.md %}
 
 ## Conveners
 
-- Meirin Oan Evans (ATLAS)
-- Michel Hernandez Villanueva (Belle II)
-- Sudhir Malik (CMS)
+- Wouter Doudeck (EIC, University of Manitoba)
```
Comment (hegner) [inline _workinggroups/training.md:154]: Deconinck

File: _workinggroups/training.md [hunk 5]
```diff
@@ -145,24 +145,23 @@ We are always looking for volunteers from the community to help us with our trai
...
+- Kilian Lieret (LMU, Belle II, *from July 2022*)
+- Michel Hernandez Villanueva (Belle II, DESY)
+- Sudhir Malik (CMS, University of Puerto Rico)
 
-The conveners can be reached [by email](mailto:michmx@phy.olemiss.edu,me338@sussex.ac.uk,malik@fnal.gov). <!-- markdown-link-check-disable-line -->
+The conveners can be [reached by email](mailto:michel.hernandez.villanueva@desy.de,wouter.deconinck@umanitoba.ca,malik@fnal.gov,kilian.lieret@posteo.de). <!-- markdown-link-check-disable-line -->
```
Comment (hegner) [inline _workinggroups/training.md:159]: Deconinck

File: _workinggroups/training.md [hunk 6]
```diff
@@ -145,24 +145,23 @@ We are always looking for volunteers from the community to help us with our trai
...
 
-{% assign persons = "Meirin Oan Evans, Sudhir Malik, Michel Hernandez Villanueva" | split: ", " %}
+{% assign persons = "Kilian Lieret, Sudhir Malik, Michel Hernandez Villanueva, Wouter Doudeck" | split: ", " %}
```
Comment (hegner) [inline _workinggroups/training.md:161]: Deconinck

**Review by hegner — CHANGES_REQUESTED** [review]

**Review by hegner — APPROVED** [review]

**Comment by klieret** [comment]
> All the link check failures seem to be the same kind as yesterday and all false positives. Maybe @klieret has an idea - it seems that `{{site.baseurl}}` is expanding to an empty string and then the URL checker is trying for an absolute path, instead of a relative one.

I've opened https://github.com/HSF/hsf.github.io/issues/1046 for this for now. Currently checking since when this is happening, will post updates in the issue.

**Pull request merged**